src/c++/perf_analyzer/test_utils.h

```diff
@@ -26,13 +26,55 @@
 
 #pragma once
 
+#include <atomic>
 #include <cmath>
 #include <cstdint>
 #include <numeric>
+#include <thread>
 #include <vector>
 
 namespace triton { namespace perfanalyzer {
 
+// FIXME
```

Comment:
Can you add a description for this fixme?

Reply:
done